### ISSUE TITLE
Lacking an author name?

### ISSUE THREAD

**Issue by minopret** [runpaint/read-ruby#132]
There does not appear to be any claim about who wrote the book. Is it satisfactory to identify the author as "runpaint.org" (alphabetize as "runpaint.org"; when necessary, enter "runpaint.org" as last name and leave first name empty)? That's what hackershelf.com did. On the other hand theassayer.org has "Run Run" as last name and "Run Paint" as first name.

**Comment by mose** [comment]
Yes this is totally disturbing. The book is good quality and it's published anonymously.

But worse even : the domain name runpaint.org, which was our only link to the author, expired 2 weeks ago and was parked by the registrar. And all we have to contact the author is that domain name now unusable. I'm afraid he won't even get any notification from github anymore if he registered with that email.

Anyone has any info on who is the author of this very good book and how to contact him/her ?

**Comment by mose** [comment]
Update : the domain name is online again ! cool ! :)

**Comment by chengcheng300** [comment]
Hi guys,

I registered a domain to continue hosting this book. Check [readruby.io](http://readruby.io).
